java-source/src/test/kotlin/net/corda/examples/obligation/contract/ObligationContractTransferTests.kt

```diff
@@ -16,7 +16,7 @@ class ObligationContractTransferTests : ObligationContractUnitTests() {
             transaction {
                 output(OBLIGATION_CONTRACT_ID) { oneDollarObligation }
                 command(ALICE_PUBKEY, BOB_PUBKEY) { DummyCommand() }
-                this `fails with` "Required ObligationContract.Commands command"
+                this `fails with` "List is empty."
```

Comment:
There are two versions of this command in the Contracts [https://github.com/corda/corda/blob/0d15e70d5199f7429894541cddbb0f8a3b381bf0/core/src/main/kotlin/net/corda/core/contracts/ContractsDSL.kt#L54](this one) which is only accessible via Kotlin and [https://github.com/corda/corda/blob/0d15e70d5199f7429894541cddbb0f8a3b381bf0/core/src/main/kotlin/net/corda/core/contracts/ContractsDSL.kt#L61](this one) which is used by the Java version. It looks like someone has improved the error message for the first function but not the second, but it looks to me as if they're the same thing.